spec/services/signups/register_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
```

Comment:
Shouldn't this be required in a more generic level?

Reply:
Files generated by Rails are like this, I just followed the pattern.